src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ManagedACTAccessRequirementWidget.java

```diff
@@ -151,10 +158,12 @@ public void onSuccess(String rootWikiId) {
     manageAccessButton.configure(ar);
     teamSubjectsWidget.configure(ar.getSubjectIds());
     entitySubjectsWidget.configure(ar.getSubjectIds());
+    accessRequirementRelatedProjectsList.configure(ar.getId().toString());
     lazyLoadHelper.setIsConfigured();
     view.setAccessRequirementID(ar.getId().toString());
     isACTMemberAsyncHandler.isACTActionAvailable(isACT -> {
       view.setAccessRequirementIDVisible(isACT);
+      view.setCoveredEntitiesHeadingVisible(isACT);
```

Comment:
Only show the "Covered Entities" heading when the projects list and entities table are shown, i.e. when the user is ACT